sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/serialization/VoidSerializer.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.beam.sdk.io.kafka.serialization;
+
+import java.util.Map;
+
+import org.apache.kafka.common.serialization.Serializer;
+
+/**
+ * Kafka {@link Serializer} for {@link Void}.
+ *
+ * <p>Will always serialize the given data as null.
+ * This is useful for {@link org.apache.kafka.clients.producer.Producer}
+ * where the key is always null.
+ */
+public class VoidSerializer<V> implements Serializer<V> {
```

Comment:
I think it is better not to have this serializer unless it is really required. It silently drops data.